CollectionRate_Analysis_CKE.txt

```diff
@@ -0,0 +1,24 @@
+The data show a slightly normal distribution throughout, which most distributions centered around a rate of 55-65.
+Ind_Team and Ind show more similar distributions, whereas Team shows a slightly more flattened distribution.
+
+Collection Rate v. Target by Strategy Plot:
```

Comment:
Before to include the type of plot (Bar plot, Interaction plot, etc.). 

Also please include which level you are referring to (i.e., Individual or Team).

Do this for all of the sections.